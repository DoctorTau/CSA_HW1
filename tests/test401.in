567
7028 -2219 7252 -2125 -9860 14 2737 3595 7344 -9153 -5715 -979 1087 1479 7682 -8230 -1722 -265 4708 431 -2182 653 -7597 4024 -6107 -3885 227 9145 -5807 7227 -9013 6129 8459 -270 -3074 5017 6568 -1627 5092 -1982 -1598 -6275 -1441 9872 -5354 318 -3169 7082 -5255 4763 8734 -1922 -8618 873 -10000 -5236 -4108 -1931 -8872 3157 5482 123 1512 -4950 -6682 5008 -1463 -1351 -1799 2254 -5555 8873 1799 -8210 8483 -7985 -3335 5868 9836 -5515 4249 3837 -1924 1637 -5314 -5927 -392 5669 5146 -193 -1987 -4788 1088 -7284 -6421 5834 -6404 4445 8469 2745 3387 -349 8921 -6474 2132 -2680 -5946 -241 1966 -3841 -367 9656 400 -618 9745 -5781 -4680 671 -8208 1573 -816 8530 2391 -5287 4047 -8924 6877 7631 9256 -1512 7316 -8620 4305 4546 -5630 -9501 739 2169 2520 9988 -3881 -540 -1592 9384 -4654 -7252 4421 1198 8829 5207 -8368 -9675 615 1970 -8735 -2273 -3546 -8799 1417 1487 3864 -2502 -4577 -4480 7713 4381 -6377 -3147 -1908 435 -2629 2289 -7502 -8001 1262 -7663 -6922 -8602 9247 207 104 -9478 -8029 -3484 6430 8083 -8690 1841 6524 399 -1669 -1069 2934 -7523 5536 -6471 2206 797 -4352 -1365 7802 34 7387 -931 5204 9868 3798 -9008 8718 4889 -9577 -162 6211 -8822 -6935 -4079 2886 278 5537 -4386 6348 2781 -5861 -1177 8563 7600 3707 2040 -3957 3004 -7061 -8152 -7390 -5285 5001 -6340 -2765 -8287 -9988 9573 -8079 -7839 -6353 -8372 -1963 -7588 -9280 4998 5113 -7027 -8317 -3046 -3468 -3246 5040 9554 -3413 2563 6714 1813 -1558 1802 -7477 4027 -2654 3898 4234 -9938 -7659 8099 -5640 -5835 -8716 5517 4993 5254 -7726 9365 3701 1643 8552 2946 -8095 6632 -1751 -4324 3882 -5704 3524 -5433 136 8558 -3318 -4273 4423 6729 6579 -7118 4694 389 8442 4507 7353 -2845 6571 3318 -6124 3265 7377 -2983 2435 2449 -3180 4560 -1606 4378 -3625 7889 -9687 -7755 -7650 -811 -3344 7155 -8630 4568 8123 7820 2873 1368 6987 -2995 6125 3469 1924 7116 -1935 9043 -6526 -855 -9256 6592 753 8219 8497 7005 5363 5752 6177 9979 -5592 -1645 -9199 4544 5566 9040 4604 3813 2671 -4441 -9338 -623 886 6341 2894 -3259 7220 204 9813 -7332 9965 -8758 -6131 1071 5411 -4957 -2726 -1573 5675 5207 4197 2558 -9807 -3247 -1707 -6584 -5134 5389 -897 -5095 -4082 7009 -1095 5755 2471 -7110 9005 -112 -6618 -2404 5013 6980 -3520 -1524 5544 -5678 -9936 3085 -6156 -6248 6177 9260 1757 9560 972 -2522 -6489 6549 -5546 -7645 -669 -2324 -1507 -1044 6788 3642 7191 -6501 -5961 5786 9590 6163 -3984 416 9643 -6665 2987 -9007 -1697 -2869 -5560 -2739 9274 -9863 524 847 5307 6353 4256 1326 6063 -958 -9603 -2256 -9768 -167 -6345 3668 9966 7370 -327 -1965 -2542 1026 -1228 6769 -8376 8455 -5450 -4716 4991 -875 2905 6072 8760 -6571 8616 8566 4070 6270 -6989 -3909 -6822 -1513 3440 6476 -798 -8490 -6780 -3680 -1659 -9472 -3410 9208 9760 -8002 7343 -5893 -9863 5851 -2759 -3473 2702 -5146 -7744 -171 -4006 9605 3333 2478 5975 -3993 -8073 -5069 -2583 -5068 -6465 -1936 -6342 -8329 -6316 9701 -1219 9470 -866 7966 6603 5575 -9105 6875 6155 5079 -7297 4969 -2156 1801 -9727 6782 -2052 1109 -4508 2498 -1182 6009 -9982 5333 -938 -6276 -9159 8289 188 6607 670 -1978 -4961 8956 -2934 -3883 -7049 7331 7731 601 2578 9399 2815 788 8788 