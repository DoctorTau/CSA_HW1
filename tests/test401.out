-9860 -8230 -1722 -2182 -270 -3074 -1982 -1598 -5354 -1922 -8618 -10000 -5236 -4108 -8872 -4950 -6682 -8210 -1924 -5314 -392 -4788 -7284 -6404 -6474 -2680 -5946 -618 -4680 -8208 -816 -8924 -1512 -8620 -5630 -540 -1592 -4654 -7252 -8368 -3546 -2502 -4480 -1908 -7502 -6922 -8602 -9478 -3484 -8690 -4352 -9008 -162 -8822 -4386 -8152 -7390 -6340 -9988 -8372 -7588 -9280 -3046 -3468 -3246 -1558 -2654 -9938 -5640 -8716 -7726 -4324 -5704 -3318 -7118 -6124 -3180 -1606 -7650 -3344 -8630 -6526 -9256 -5592 -9338 -7332 -8758 -2726 -6584 -5134 -4082 -7110 -112 -6618 -2404 -3520 -1524 -5678 -9936 -6156 -6248 -2522 -5546 -2324 -1044 -3984 -5560 -958 -2256 -9768 -2542 -1228 -8376 -5450 -4716 -6822 -798 -8490 -6780 -3680 -9472 -3410 -8002 -5146 -7744 -4006 -5068 -1936 -6342 -6316 -866 -2156 -2052 -4508 -1182 -9982 -938 -6276 -1978 -2934 